packages/newspringchurchapp/src/prayer/PrayerList/index.js

```diff
@@ -109,72 +133,82 @@ class PrayerList extends PureComponent {
+                        };
+                        await cache.writeQuery({
+                          query,
+                          data: newPrayerObject,
+                        });
+                      }}
+                    >
+                      {(flagPrayer) => (
+                        <PrayerCardConnected
+                          avatarSize={'medium'}
+                          expanded
+                          actionsEnabled
+                          navigation={navigation}
+                          cardIndex={item.index}
+                          prayerId={item.item.id}
+                          advancePrayer={this.scrollToNext}
+                          options={[
+                            {
+                              title: 'Flag as Inappropriate',
+                              method: async () => {
+                                await flagPrayer({
+                                  variables: {
+                                    parsedId: item.item.id,
+                                  },
+                                });
+                              },
+                              destructive: true,
                             },
-                            destructive: true,
-                          },
-                        ]}
-                        {...item.item}
-                      />
-                    )}
-                  </Mutation>
-                )}
-                ItemSeparatorComponent={() => (
-                  <DividerView>
-                    <GreyH6>Press down on card to pray</GreyH6>
-                  </DividerView>
-                )}
-                data={get(data, prayers, []).map((prayer) => ({
-                  key: prayer.id,
-                  id: prayer.id,
-                  prayer: prayer.text,
-                  source: prayer.campus.name || '',
-                  name: prayer.firstName,
-                }))}
-                scrollEnabled={false}
-              />
-            )}
+                          ]}
+                          {...item.item}
+                        />
+                      )}
+                    </Mutation>
+                  )}
+                  ItemSeparatorComponent={() => (
+                    <DividerView>
+                      <GreyH6>Press down on card to pray</GreyH6>
+                    </DividerView>
+                  )}
+                  data={get(data, prayers, []).map((prayer) => ({
+                    key: prayer.id,
+                    id: prayer.id,
+                    prayer: prayer.text,
+                    source: prayer.campus.name || '',
+                    name: prayer.firstName,
+                  }))}
+                  scrollEnabled={false}
+                />
+              ) : (
+                <StyledView>
+                  <StyledH3View>
+                    <H3>No Prayers!</H3>
+                  </StyledH3View>
+                  <BodyText>{noPrayerText}</BodyText>
```

Comment:
```suggestion
                  <BodyText>{`There are currently no prayers for your ${type}. Go back and add one!`}</BodyText>
```

Reply:
the variable here is prob unnecessary here since we're just hardcoding a few lines up

Reply:
Haha! I tried this first and couldn't get it to work so that's why I made the variable. I think I was missing the `{}` around it the first time. I totally added this. Great catch! 🎉